db/migrate/20181123103416_rename_attachment_on_s3_to_attachment_for_all_hosted_files.rb

```diff
@@ -0,0 +1,18 @@
+class RenameAttachmentOnS3ToAttachmentForAllHostedFiles < ActiveRecord::Migration[5.2]
+  def change
+    %w(image video audio text_track).each do |hosted_file_type|
+      rename_column "pageflow_#{hosted_file_type}_files",
+                    :attachment_on_s3_file_name,
+                    :attachment_file_name
+      rename_column "pageflow_#{hosted_file_type}_files",
+                    :attachment_on_s3_content_type,
+                    :attachment_content_type
+      rename_column "pageflow_#{hosted_file_type}_files",
+                    :attachment_on_s3_file_size,
+                    :attachment_file_size
+      rename_column "pageflow_#{hosted_file_type}_files",
+                    :attachment_on_s3_updated_at,
+                    :attachment_updated_at
```

Comment:
Is `file_size`/`content_type` still set? 

We still need think about migrating `pageflow-panorama`.

Reply:
`file_size` yes, `content_type` no, apparently. But as discussed, I'll remove this migration for compatibility reasons (S3 bucket name)